src/Routes.js

```diff
@@ -91,81 +74,18 @@ const TemplateDetail = lazy(() =>
 );
 
 export const Routes = () => {
-    const history = useHistory();
+    const navigate = useNavigate();
     const chrome = useChrome();
 
     const generalPermissions = ['patch:*:*', 'patch:*:read'];
     const [hasSystems, setHasSystems] = useState(true);
     const INVENTORY_TOTAL_FETCH_URL = '/api/inventory/v1/hosts';
 
-    const paths = [
-        {
-            path: '/advisories/:advisoryId/:inventoryId',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: InventoryDetail
-        },
-        {
-            path: '/advisories/:advisoryId',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: AdvisoryPage
-        },
-        {
-            path: '/advisories',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: Advisories
-        },
-        {
-            path: '/systems/:inventoryId',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: InventoryDetail
-        },
-        {
-            path: '/systems',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: Systems
-        },
-        {
-            path: '/packages/:packageName/:inventoryId',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: InventoryDetail
-        },
-        {
-            path: '/packages/:packageName',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: PackageDetail
-        },
-        {
-            path: '/packages',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: PackagesPage
-        },
-        {
-            path: '/templates/:templateName',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: TemplateDetail
-        },
-        {
-            path: '/templates',
-            isExact: true,
-            requiredPermissions: generalPermissions,
-            component: Templates
-        }
-    ];
-
     const listenNavigation = useCallback(() => {
         if (chrome) {
             return chrome.on('APP_NAVIGATION', event => {
                 if (event.domEvent) {
-                    history.push(`/${event.navId}`);
+                    navigate(`/${event.navId}`);
```

Comment:
I think, this should use `chrome.appNavClick` like here[0] in order to work correctly. We have encountered with navigate where it does a full page refresh, which it shouldn't.

[0] https://github.com/RedHatInsights/compliance-frontend/pull/1892/files#diff-3d74dddefb6e35fbffe3c76ec0712d5c416352d9449e2fcc8210a9dee57dff67R23